File: 97 Large non-Mersenne prime/li.c
#include <stdio.h>
#include <math.h>
#include <time.h>

int main() {
	clock_t start = clock();
    long n;
    long p, b=1;
    p = 7830457;
    for( n=28433 ; b<= 7830457; b++ ){
        n = n * 2 % 10000000000;
    }
    n++;
	clock_t finish = clock();
    double duration = (double)(finish - start) / CLOCKS_PER_SEC;
	printf("%ld\n", n);
    printf( "%f seconds\n", duration );
	
}

File: 97 Large non-Mersenne prime/li.js
start = new Date().getTime();
for(i=0,tmp=28433;i<7830457;++i) tmp = tmp * 2 % 10000000000; tmp++;
result = tmp;
end = new Date().getTime();
console.log("result:", result, "time:", (end - start)/1000 + "s")
//result: 8739992577 time: 0.129s